pkg/kubecost/summaryallocation.go

```diff
@@ -830,30 +825,30 @@ func (sas *SummaryAllocationSet) AggregateBy(aggregateBy []string, options *Allo
 			// Percentage of idle that should remain after filters are applied,
 			// which equals the proportion of filtered-to-actual cost.
 			cpuFilterCoeff := 0.0
-			if allocTotals[key].CPUCost > 0.0 {
+			if allocTotals[key].TotalCPUCost() > 0.0 {
 				filteredAlloc, ok := allocTotalsAfterFilters[key]
 				if ok {
-					cpuFilterCoeff = filteredAlloc.CPUCost / allocTotals[key].CPUCost
+					cpuFilterCoeff = filteredAlloc.TotalCPUCost() / allocTotals[key].TotalCPUCost()
```

Comment:
This change (and the corresponding GPU and RAM changes) is the one that fixes the issue. In fact, only `allocTotals[key].TotalCPUCost()` is the fix -- the other one won't ever have adjustments factored in, but using `TotalCPUCost()` is just as correct.